workinggroups/serverless.md

```diff
@@ -0,0 +1,39 @@
+# CNCF Serverless WG Proposal
+
+## TOC Sponsor
+
+Ken Owens
+
+## Objective
+
+Explore the intersection of cloud native and serverless technology.
+
+## Goals and Expected Outcomes
+
+* Produce a whitepaper
+* Produce a serverless landscape
+* Explore specifications for serverless to propose to the CNCF
+* Bring recommendations to the TOC on serverless projects in CNCF
```

Comment:
This may be a bit of a sidebar but, is the CNCF looking to bring projects of certain types into its fold? Or, would it rather projects approach them? I'm curious on this point.